test/utils/updateRateOracle.ts

```diff
@@ -0,0 +1,20 @@
+import { TestTimeAveragedBaseRateOracle, TimeAveragedBaseRateOracle } from 'contracts'
+import { MockProvider } from 'ethereum-waffle'
+import { DAY, timeTravelTo } from '.'
+
+export const updateRateOracle = async (oracle: TimeAveragedBaseRateOracle | TestTimeAveragedBaseRateOracle, cooldown: number, provider: MockProvider) => {
```

Comment:
I like that this function is extracted here. However I just noticed one possible issue: when there is `timeTravel()` called independently from this function it is possible that `timeTravelTo()` here will bring us to the past thus causing an error. `timeTravelTo` only if the time we are traveling to is greater than `now`.

Reply:
I honestly thought of it, but I really think that the person writing tests should be aware of it, and simply call `oracle.update()` if the time has passed